app/src/main/res/layout/fragment_select_flavor_bottom_sheet.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent" android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/tvInfo"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Please select the best category."
```

Comment:
Can you extract the string ?

Reply:
Missed that one and one more thing, I couldn't find the respective icons for the category. Can you help me?